src/plugins/data/public/ui/dataset_selector/configurator.tsx

```diff
@@ -165,6 +192,7 @@ export const Configurator = ({
             onConfirm(dataset);
           }}
           fill
+          disabled={timeFieldName === undefined && requiresTimeField(dataset, language, timeFields)}
```

Comment:
this might be an issue because we don't want to query the dataset source with a string it should be undefined